src/Microsoft.Extensions.Logging/Logger.cs

```diff
@@ -9,40 +9,13 @@ namespace Microsoft.Extensions.Logging
 {
     internal class Logger : ILogger
     {
-        private readonly LoggerFactory _loggerFactory;
-
-        private LoggerInformation[] _loggers;
-
-        private int _scopeCount;
-
-        public Logger(LoggerFactory loggerFactory)
-        {
-            _loggerFactory = loggerFactory;
-        }
-
-        public LoggerInformation[] Loggers
-        {
-            get { return _loggers; }
-            set
-            {
-                var scopeSize = 0;
-                foreach (var loggerInformation in value)
-                {
-                    if (loggerInformation.CreateScopes)
-                    {
-                        scopeSize++;
-                    }
-                }
-                _scopeCount = scopeSize;
-                _loggers = value;
-            }
-        }
-
-        public bool CaptureScopes { get; set; }
+        public LoggerInformation[] Loggers { get;set; }
```

Comment:
nit: space after `;`

Reply:
Is there a good reason these are array instead of `List<...>`? We could save calling `.ToArray()` if they were `List<...>`

Reply:
Most of the time (when ILoggerFactory,AddProvider is no used) this would get set only once on startup, seemed wasteful to keep the list around.